MvvmCross.Forms/Platform/iOS/Views/MvxFormsIosViewPresenter.cs

```diff
@@ -90,7 +90,7 @@ public virtual bool ShowPlatformHost(Type hostViewModel = null)
         public virtual bool ClosePlatformViews()
         {
             CloseMasterNavigationController();
-            CleanupModalViewControllers();
+            CloseModalViewControllers();
```

Comment:
Same about this method, maybe we can keep both and mark the old one as obsolete? The old one should only call the new one